### PR TITLE
[Carbon Black Enterprise Response] New pack Carbon Black Common Fields for Unique ID field

### PR DESCRIPTION
**Pull request by ennozdd**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/content/pull/19834

## Description
Unique ID is needed to update the alerts. With this PR Carbon Black EDR Unique ID is created and mapped

https://developer.carbonblack.com/reference/enterprise-response/5.1/rest-api/#updateresolve-alerts

Edit: On my latest PR I was asked to create another pack. See https://github.com/demisto/content/pull/19834

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Hi, thank you for this contribution!
Before we start the review, could you please explain why you added a whole new pack for the new incident field? It can be added to the pack itself (carbon_Black_Enterprise_Response) instead.

**Review by idovandijk — APPROVED** [review]
nice job!

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Great job.
One thing to fix - No release note entry was found for the mapper "Carbon Black EDR Mapper" in the Carbon_Black_Enterprise_Response pack. Please add it.

File: .circleci/config.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ references:
 
   container_config: &container_config
     docker:
-      - image: devdemisto/content-build:3.0.0.33772  # disable-secrets-detection
```
Comment (JasBeilin) [inline .circleci/config.yml:97]: Why are we using devdemisto docker image?

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Looks good, lets change the docker image to a production one and merge :)

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**